packages/subapp-pbundle/lib/framework-lib.js

```diff
@@ -169,8 +170,16 @@ class FrameworkLib {
 
   async realizeReduxStore() {
     if (this.store && this.store.realize) {
+      const { subApp, subAppServer } = this.ref;
       this.store = this.store.realize();
       await this.signalStoreReady();
+
+      const packReduxData = subAppServer.packReduxData || subApp.packReduxData;
+      // if subapp didn't request to skip sending initial state and packReduxData was specified,
+      //  then stringify packReduxData's return value and attach it to the index html.
+      if (packReduxData && subAppServer.attachInitialState !== false) {
+        this.initialStateStr = JSON.stringify(packReduxData(this.store));
+      }
```

Comment:
this should be done outside of the if, because a store may have been realized already.

Also, only do this if `this.initialStateStr` is falsy